test/server/graph/context.js

```diff
@@ -1,8 +1,8 @@
 const expect = require('chai').expect;
 
-const User = require('../../models/user');
-const Context = require('../../graph/context');
-const errors = require('../../errors');
+const User = require('../../../models/user');
```

Comment:
It's possible to use `https://www.npmjs.com/package/babel-plugin-module-alias` or `https://www.npmjs.com/package/babel-plugin-webpack-alias` which would require `babel-node`.